WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -103,6 +103,8 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
         return indicator
     }()
 
+    private let shouldPushDetailsViewController = UIDevice.current.userInterfaceIdiom != .pad
```

Comment:
I think this check doesn't take into consideration when the app is in split mode.

![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/bb1ed31b-55fd-4594-8fa1-34d1d452028b)

Reply:
Good catch, Salim!

Reply:
I see that our app does not react on activating the split screen mode and it's worth creating a separate issue for that. You can check `My Site` -> `Media` flow:

![simulator_screenshot_1432A8E7-D7CF-46BD-8121-52C0DC370B90](https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/83238ff8-6152-4d24-be59-9a46b937e2bc)

Reply:
I suspect that using `UIDevice.device` might not be the best approach to determine whether to hide the bottom bar. This is because, on iPad devices, the app window can be resized to resemble that of an iPhone during split mode.

> it's worth creating a separate issue for that.

Sure thing, this issue is not blocking.

Reply:
> I suspect that using UIDevice.current might not be the best approach to determine whether to hide the bottom bar. This is because, on iPad devices, the app window can be resized to resemble that of an iPhone during split mode.

Yeah, totally agree! I'm planning to fix such issues in the next steps and solve this problem for "My Site" content too (such as "Media" screen)